mitmproxy/ctxmanager.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import, print_function, division
+import six
+
+
+class Facade(object):
+    def __init__(self, getobj):
+        self._getobj = getobj
+
+    if six.PY3:
```

Comment:
Using `six.PY3` is generally not a good idea - once Py4 is here, this will break...